MusicNotationKit/MusicNotationKit/Key.swift

```diff
@@ -18,3 +18,11 @@ public struct Key {
 		self.type = type
 	}
 }
+
```

Comment:
Add `// MARK: - Protocols` and then underneath it: `// MARK: CustomDebugStringConvertible`.

See the SlideShare Swift style guide for explanations.